templates/experiments/experiment_form.html

```diff
@@ -81,6 +81,18 @@ <h3 class="text-md font-bold">{% translate "Please note" %} <i class="fas fa-exc
         {% render_form_fields form "tools" %}
       </div>
 
+      <input type="radio" name="main_tabs" role="tab" class="tab" aria-label="Allowlist">
+      <div role="tabpanel" class="tab-content">
+        <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
+        <p class="py-4 text-sm">
+          {% blocktranslate %}
+            To limit the set of particiapants that can chat with this bot, simply add their identifiers in this list.
+            When the list is empty, the bot will be public.
```

Comment:
I don't think users will know what being public means. Maybe reword to this:
```suggestion
            When the list is empty, the bot will accessible via its public link.
```